insights/parsers/alternatives.py

```diff
@@ -151,3 +151,27 @@ class JavaAlternatives(AlternativesOutput):
         '/usr/lib/jvm/jre-1.6.0-ibm.x86_64/bin/ControlPanel'
     """
     pass
+
+
+@parser(Specs.alternatives_display_python)
+class PythonAlternatives(AlternativesOutput):
+    """
+    Class to read the ``/usr/sbin/alternatives --display python`` output.
+
+    Uses the ``AlternativesOutput`` base class to get information about the
+    alternatives for ``best`` available and which one is currently in use.
+
+    Examples:
+        >>> python = shared[AlternativesOutput]
```

Comment:
@JoySnow the `shared[...]` method is deprecated and no longer valid, please update your documentation and add testing for your docstrings.  See the [documentation](https://insights-core.readthedocs.io/en/latest/docs_guidelines.html) for more information.

Reply:
Hi @bfahr, Thanks for the info. The docstrings part is enhanced. Please kindly review.